unattended_scripts/install_functions/opendistro/elasticsearch.sh

```diff
@@ -78,6 +78,7 @@ configureElasticsearchAIO() {
 
     eval "cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ ${debug}"
     eval "./securityadmin.sh -cd ../securityconfig/ -icl -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin-key.pem ${debug}"
+    cd -
```

Comment:
It should go inside an eval "... $ {debug}" if possible, this causes the path to be shown during the installation, since `cd -` changes to the previous directory and shows it on the screen